elasticdl/Makefile

```diff
@@ -1,21 +1,22 @@
 PYTHON = python
-
-EDL_PROTO_FILE = elasticdl/proto/elasticdl.proto
-
+PROTO_PATH=elasticdl/proto
 GO_FILE_DIR = elasticdl/pkg/proto
-GO_PB_FILE = $(GO_FILE_DIR)/elasticdl.pb.go
-
 CXX = g++
 CAPI_DIR = elasticdl/pkg/kernel/capi
 
-all: python_pb $(GO_PB_FILE) kernel
+all: python_pb go_pb kernel
 
 python_pb:
```

Comment:
This target requires at least ${PROTO_PATH}/elasticdl.proto as its dependent.

Reply:
Done